lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
I thought so, too, but:

    $ mkdir true && /usr/bin/env bash -c 'compgen -A command -o dirnames true'
    true
    true
    true

- builtin
- /bin/true
- directory

Also note that the directory does *not* get a slash via bashcomp. Apparently this is done by readline afterwards ... I haven't found a way around this yet. Maybe we can somehow interact with readline in a non-interactive context, but that will involve more research and testing. But anyways, I'm afraid for the moment we cannot ditch the `sort -u` ...